src/components/HomePage/SearchResultsArea/Offer/OfferApplyButton.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle, Grid, makeStyles, Tooltip } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+    dialog: {
+        padding: theme.spacing(2),
+        textAlign: "center",
+    },
```

Comment:
Maybe we could refactor and create our custom dialog component and use them both here and [here](https://github.com/NIAEFEUP/nijobs-fe/blob/1e91c49fa197cc1968f0c1fa54bb3e933987f073/src/cookieConsent.js)
What are your thougths?